### PR TITLE
IGVF-1051-fix-crispr-audit

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/audit/measurement_set.py [hunk 1]
```diff
@@ -96,8 +96,9 @@ def audit_inconsistent_readout(value, system):
     assay = system.get('request').embed(assay_term, '@@object?skip_calculated=true')
     screen_assays = ['Perturb-seq',
                      'CRISPR screen',
-                     'massively parallel reporter assay']
-    if assay.get('term_name') in screen_assays:
+                     'massively parallel reporter assay',
+                     'cas mediated mutagenesis']
+    if assay.get('term_name') in screen_assays or value.get('preferred_assay_title', '') in screen_assays:
```
Comment (gabdank) [inline src/igvfd/audit/measurement_set.py:101]: MPRA is not a screen assay, i think we need to rename the list to be "assays_with_readout" or something like that

File: src/igvfd/audit/measurement_set.py [hunk 2]
```diff
@@ -134,25 +135,22 @@ def audit_inconsistent_modifications(value, system):
         yield AuditFailure('inconsistent modifications', detail, level='NOT_COMPLIANT')
 
 
-@audit_checker('MeasurementSet', frame='object')
+@audit_checker('MeasurementSet', frame='embedded')
```
Comment (gabdank) [inline src/igvfd/audit/measurement_set.py:138]: we should not rely on embedding in my opinion

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/igvfd/audit/measurement_set.py (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/audit/measurement_set.py]: reverted to object

File: src/igvfd/audit/measurement_set.py (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/audit/measurement_set.py]: reworded

**Review by gabdank — APPROVED** [review]

**Review by gabdank — APPROVED** [review]

**Pull request merged**